src/components/settings/supportFerdi/SupportFerdiDashboard.tsx

```diff
@@ -29,50 +29,50 @@ const SupportFerdiDashboard = () => {
         <div>
           <p className="settings__support-badges">
             <a
-              href="https://twitter.com/getferdi/"
+              href="https://twitter.com/ferdium/"
```

Comment:
```suggestion
              href="https://twitter.com/ferdiumteam/"
```

I don't know if the twitter account appears somewhere else